.github/workflows/deploy.yml

```
@@ -0,0 +1,81 @@
+name: Deploy to Azure
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)